Rakefile

```diff
@@ -6,3 +6,9 @@ Rake::TestTask.new do |t|
   t.libs << "test"
   t.pattern = "test/**/*_test.rb"
 end
+
+task :test2
+Rake::TestTask.new do |t|
+  t.libs << "test"
+  t.pattern = "test/**/*_test_citus.rb"
+end
```

Comment:
In unix based systems, files should have one blank line at the end. This weird symbol indicates that there is no blank line.